datalake-dr-connector/build.gradle

```diff
@@ -0,0 +1,55 @@
+apply plugin: 'com.google.protobuf'
+
+buildscript {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        maven { url = "$repoUrl" }
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.8'
+    }
+}
+
+dependencies {
+    compile project(':common')
+    compile project(':grpc-common')
+    compile project(':auth-connector')
+    compile project(':structuredevent-model')
+    compile project(':flow')
```

Comment:
are you sure these dependencies are necessary? I'm not sure about you need common here

Reply:
I've cleaned up the build.gradle file to remove any unnecessary dependencies. I did need common for the logger, but there was a bunch of other stuff I could take out. I've also updated the header to "CB-8717 Add thunderhead datalake-dr service connector".